### ISSUE TITLE
🐧 splahlunking ↔ v23.4.1

### ISSUE THREAD

**Issue by sneslink** [itchio/itch-compatibility-watchlist#1014]
:rotating_light: game [splahlunking](https://xhg.itch.io/splahlunking) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/a4eb089d1b9f26af748d32398d11ee65).

:running: the Itch.io app appears to be attempting to run a .love file as a binary on the Linux version

**Comment by fasterthanlime** [comment]
Yep, that's because the .love file is tagged as a Linux binary (by the developer) even though, well, it isn't :(

The plan is to teach the itch.io app to deal with .love files properly though. That'll come :)
